deployment/cloud-run/test_complete_api.py

```
@@ -0,0 +1,318 @@
+
+        # Use early return pattern to avoid nested conditionals
+        if response.status_code != 200:
+            print(f"   ❌ Failed - {name}")
+            print(f"   Response: {response.text[:200]}...")
+            return False, response.text
+
+        # Success case
+        try:
+            data = response.json()
+            print(f"   ✅ Success - {name}")
+            return True, data
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)